### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by marijanp**
Closes # .

### Pre-review checklist

- [ ] All code has been formatted using our config (`make format`)
- [ ] Any new API features or modification of existing behavior is covered as defined in our [test plan](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/test-plan.md)
- [ ] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 
+- added class instances:
```
Comment (klntsky) [inline CHANGELOG.md:36]: This is not something the user can understand if not very familiar with CTL. Please describe the motivation for the change, and not the implementation. I think inserting a permanent link to the example code would be helpful too. Don't mention the tests (they are mandatory anyway).

**Review by klntsky — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,6 +33,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 
+- Support passsing the inital UTxO distribution as an Array and also get the KeyWallets as an Array when writing Plutip tests.
+To achieve this the following class instances were added ([#1018](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1018)):
+  - `UtxoDistribution (Array InitialUTxOs) (Array KeyWallet)`
+  - `UtxoDistribution (Array InitialUTxOsWithStakeKey) (Array KeyWallet)`
+
+  Example usage:
```
Comment (klntsky) [inline CHANGELOG.md:41]: Sorry for bothering you again. A usage example would be fit in `doc/plutip-testing.md` (a link here is OK). The changelog is needed to help the users update quickly, so it must be succinct.

**Review by klntsky — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (klntsky) [reply CHANGELOG.md:41]: Please remove the mentions of UtxoDistribution.

Comment (marijanp) [reply CHANGELOG.md:41]: there is already an usage example there, I've added the link and removed the mentions

**Review by klntsky — APPROVED** [review]

**Pull request merged**